awx/api/serializers.py

```diff
@@ -1280,6 +1280,14 @@ def get_summary_fields(self, obj):
                     'job_templates': 0, 'admins': 0, 'projects': 0}
             else:
                 summary_dict['related_field_counts'] = counts_dict[obj.id]
+
+        # Organization participation roles (admin, member) can't be assigned
+        # to a team. This provides a hint to the ui so it can know to not
+        # display these roles for team role selection.
+        for key in ('admin_role', 'member_role',):
+            if key in summary_dict['object_roles']:
```

Comment:
Does `['object_roles']` exist in all contexts?  Might be more defensive to do `.get('object_roles', {})`

Reply:
It's probably fine, that said there's no harm in adding a `.get` just in case.

Reply:
done.